scenarioo-client/app/search/search.html

```diff
@@ -25,7 +25,7 @@
      data-treedata="vm.results.resultSet.children"
      data-treemodel="vm.treemodel"
      data-click-action="vm.goToRelatedView(node)"
-     data-first-column-title="Search Results"
+     data-first-column-title="Search Results (showing {{vm.hits}} of {{vm.totalHits}} hits)"
```

Comment:
@bruderol Yes, as soon as there are more than 200 results, this is visible here. I tested it with setting the limit to 20:
![image](https://cloud.githubusercontent.com/assets/3780183/20652338/bf189328-b4f7-11e6-8127-1937abc141cd.png)